server/models/migrations/next/migrate.sql

```diff
@@ -303,6 +303,15 @@ INSERT IGNORE INTO `report` (`report_key`, `title_key`) VALUES
 CALL drop_foreign_key('shipment_item', 'shipment_item__condition');
 CALL drop_column_if_exists('shipment_item', 'condition_id');
 
+/**
+ * Convert asset_condition to constants
+ * @author: jmcameron
+ * @date: 2022-05-05
+ */
+
+DROP TABLE asset_scan;
```

Comment:
Instead of adding, then dropping the table, let's just remove the add table from this migrate file.  Since the release hasn't been published yet, we don't need to add then drop.

It's found up here:

https://github.com/IMA-WorldHealth/bhima/pull/6597/files#diff-2a7be2572b29530351a6070a44834c3cdae1a97785db7ed3010d615cb36ec358L222